test/demo/contextListener.test.ts

```diff
@@ -0,0 +1,46 @@
+import 'jest';
+import * as fdc3Remote from './utils/fdc3RemoteExecution';
+import {getFinConnection} from './utils/fin';
+
+
+describe('advanced puppeteer functionality', () => {
+    const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+    const testIdentity1 = {uuid: 'test-app-1', name: 'test-app-1'};
+    const testIdentity2 = {uuid: 'test-app-2', name: 'test-app-2'};
+
+    it('can can register and trigger a context listener', async () => {
+        // Creating the apps can take a while, so we need a longer timeout.
+        jest.setTimeout(10000);
+
+        const fin = await getFinConnection();
+
+        // Check to see that the main window started correctly
+        expect(fin.Application.wrapSync(testManagerIdentity).isRunning()).resolves.toBe(true);
+
+        // Launch two test applications
+        await fdc3Remote.open(testManagerIdentity, testIdentity1.uuid);
+        await fdc3Remote.open(testManagerIdentity, testIdentity2.uuid);
+
+        // Check that the apps have started properly
+        expect(fin.Application.wrapSync(testIdentity1).isRunning()).resolves.toBe(true);
```

Comment:
If we're using `expect(...).resolves`, should that be prefixed with await?

Reply:
Good catch. Will update